script/update-content-versions-and-check-for-archived-slugs.ts

```diff
@@ -50,10 +44,21 @@ async function updateRedirections(_bookId: string, currentVersion: string, _newV
       throw error;
     });
 
+  const redirectsBookPath = path.resolve(redirectsPath, bookId + '.json');
+
+  try {
+    // Create file with empty array only if it does not exixts
+    fs.writeFileSync(redirectsBookPath, '[]', { encoding: 'utf8', flag: 'wx' });
+  } catch {
+    // This will throw if file exists
+  }
+
+  const redirects: Redirects = require(redirectsBookPath);
+
   const flatCurrentTree = flattenArchiveTree(currentTree);
 
   const findRedirect = (section: LinkedArchiveTreeNode) => (
-    { pageId, bookId: pageBookId, pathname }: Redirect
+    { pageId, bookId: pageBookId, pathname }: Redirects[0]
```

Comment:
karina and i discovered that you should use `Redirects[number]` here, or ts might try to be clever and extract the first element of a tuple